pkg/ebpf/offsetguess.go

```diff
@@ -359,6 +375,7 @@ func setReadyState(m *elf.Module, mp *elf.Map, status *tracerStatus) error {
 // guess the next field.
 func guessOffsets(m *elf.Module, cfg *Config) error {
 	mp := m.Map(string(tracerStatusMap))
+	threshold = cfg.OffsetGuessThreshold
```

Comment:
Let's pass the threshold into `checkAndUpdateCurrentOffset` instead of storing threshold as a global variable.